sentry-ruby/lib/sentry/rack/capture_exceptions.rb

```diff
@@ -17,23 +17,23 @@ def call(env)
           scope.set_rack_env(env)
 
           transaction = start_transaction(env, scope)
-          scope.set_span(transaction)
+          transaction ? scope.set_span(transaction) : nil
```

Comment:
since we don't need the return value from this line, it's not necessary to return the `nil`. this should do the work too

```ruby
scope.set_span(transaction) if transaction
```

Reply:
So is that formulation more idiomatic for Ruby? As someone coming from Python and JS/TS, having the condition after the action made my brain hurt a little, which is why I went with the ternary in the end. But if the `do x if y` pattern is the accepted way, that's good to know. 🙂

Reply:
yeah this is a common practice in Ruby 😂 we're weirdos 😉